src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Handlers/ConnectionHandlerContext.cs

```diff
@@ -31,5 +31,40 @@ internal class ConnectionHandlerContext : HandlerRequest
         /// Class used by orchestrator while chaining handlers.
         /// </summary>
         public Exception Error { get; set; }
+
+        /// <summary>
+        /// Method needed to clone ConnectionHandlerContext as part of history along the chain.
+        /// </summary>
+        public object Clone() 
```

Comment:
What is the purpose of cloning a context?

Reply:
I think this shouldn't be relevant anymore, based on last discussions, as we don't aim to keep clones of context, but only history of redirected endpoints in a specialized data object.